src/libraries/System.Runtime.InteropServices/tests/ComDisabled/System/Runtime/InteropServices/ComAwareEventInfoTests.Windows.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Runtime.InteropServices.Tests.Common;
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public partial class ComAwareEventInfoTests
+    {
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void AddEventHandler_ComObjectWithMultipleComEventInterfaceAttribute_ThrowsAmbiguousMatchException()
+        {
+            Assert.Throws<NotSupportedException>(() => new ComImportObject());
```

Comment:
This test isn't doing anything with `ComAwareEventInfo`. It is more like `InstantiateComImportClass_ThrowsNotSupportedException`

We shouldn't need specific tests for `ComAwareEventInfo` (disabling built-in COM should just mean that the type it is given will never be a COM object).